packages/devtools_app/lib/src/shared/service_registrations.dart

```diff
@@ -19,11 +19,11 @@ class RegisteredServiceDescription {
     this.gaItem,
   });
 
-  final String service;
-  final String title;
-  final Widget icon;
-  final String gaScreenName;
-  final String gaItem;
+  final String? service;
+  final String? title;
```

Comment:
let's make `service` and `title` required and non-nullable

Reply:
done